.github/20-feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+
+---
+
+<em>tag:feature_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using) :
+- Are you willing to contribute it (Yes/No) :
```

Comment:
This isn't really "System information" - could it be included in the questions below?